srcpkgs/efibootmgr/files/README.voidlinux

```diff
@@ -2,8 +2,8 @@ efibootmgr can automatically generate EFI boot entries to
 directly boot the kernel after every kernel update or installation.
 Enable this by editing /etc/default/efibootmgr-kernel-hook.
 
-The EFI variables need to be accessible in order for this to work, so
-add this line to your fstab and reboot or mount manually:
+The EFI variables need to be accessible in order for this to work, make sure
+efivarfs is mounted or add this line to your fstab and reboot or mount:
```

Comment:
@ahesford Are you happy with the rewording of the text.

Reply:
Other than needing a "so" after the comma and before "make sure efivarfs", this rewording is fine.